### ISSUE TITLE
add the reload button that is available only for localhost also to other environments

### ISSUE THREAD

**Issue by Slackman2015** [storyblok/storyblok#612]
**The feature would affect:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**
When working on the localhost in the development phase it is possible to reload only the in the iframe running app: https://www.storyblok.com/cl/reload-button-for-dev
In other environments, It is necessary to reload the complete page when something in the app's code change as the reload button isn't available there.

**Describe the solution you'd like**
It would be really helpful to be able to reload only the webpage in the iframe instead of the complete editor when developing in other environments.

**Comment by LaurenceSime** [comment]
That would be very helpful.

**Comment by cartok** [comment]
Yes agree! For us it's even more important because we need to reload the page after resizing to get a properly rendered result. Thats because we ship different documents depending on the user-agent that we mock in storyblok by checking the window size.

**Comment by alvarosabu** [comment]
Hi @Slackman2015, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
